### DISCUSSION TITLE
2021-12-11-mermaid-class-diagram

### DISCUSSION THREAD

**Discussion by giscus** [PasseRR/passerr.github.io#14] [Announcements]
# 2021-12-11-mermaid-class-diagram

PasseRR's Blog

https://www.xiehai.zone/2021-12-11-mermaid-class-diagram.html

**Comment by ATang0729** [comment]
兄弟，你最后的类图和mermaid的代码没对上，你的类图里鸟翅膀的基数怎么可以是“0..2”哈哈哈哈你是疯掉了吗

**Reply by PasseRR** [reply]
感谢指出错误，已修改。

**Comment by ZYKWLJ** [comment]
写的真好！
